README.md

```diff
@@ -35,33 +34,40 @@ At a high level, this is what Quicktest is all about:
 
 ## Quick Start
 
-**Quicktest** requires [quickemu](https://github.com/quickemu-project/quickemu) (and quickget), `qemu`, `ffmpeg`, `socat`, `convert` (from imagemagick) and `tesseract`
+**Quicktest** requires [quickemu](https://github.com/quickemu-project/quickemu) (and quickget), `qemu`, `ffmpeg`, `socat`, `convert` (from imagemagick) and `tesseract` : e.g.
 
-e.g. `sudo apt install socat ffmpeg imagemagick tesseract`
+``` shell
+sudo apt install socat ffmpeg imagemagick tesseract
```

Comment:
```suggestion
sudo apt install socat ffmpeg imagemagick tesseract-ocr
```
Or not

Reply:
You're right!

Reply:
or wrong ...  There should have been a button for you to commit that suggestion to my branch if you liked it btw.